### PR TITLE
[New Rule] Persistence via MS Office Addins

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/379

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
Example of Alerts:

image

File: rules/windows/persistence_ms_office_addins_file.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+ wildcard(file.extension,"wll","xll","ppa","ppam","xla","xlam") and
+ wildcard(file.path, "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Word\\Startup\\*",
+                      "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\AddIns\\*",
+                       "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Excel\\XLSTART\\*")
```
Comment (threat-punter) [inline rules/windows/persistence_ms_office_addins_file.toml:27]:
```suggestion
file where event.type != "deletion" and
 wildcard(file.extension,"wll","xll","ppa","ppam","xla","xlam") and
 wildcard(file.path, "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Word\\Startup\\*",
                     "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\AddIns\\*",
                     "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Excel\\XLSTART\\*")
```

**Review by threat-punter — APPROVED** [review]
👍

**Review by bm11100 — APPROVED** [review]
⭐

**Pull request merged**